t/21_sles4sap_azure_cli.t

```diff
@@ -649,4 +649,67 @@ subtest '[az_network_peering_delete]' => sub {
     ok((any { /az network vnet peering delete/ } @calls), 'Correct composition of the main command');
 };
 
+
+subtest '[az_disk_create] Create disk by cloning' => sub {
+    my $azcli = Test::MockModule->new('sles4sap::azure_cli', no_auto => 1);
+    my @calls;
+    $azcli->redefine(assert_script_run => sub { @calls = $_[0]; return; });
+
+    az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka', source => 'Harvepino');
+    note("CMD: " . join(' ', @calls));
+    ok(grep(/az disk create/, @calls), 'Test base command');
+    ok(grep(/--resource-group Pa_a_Pi/, @calls), 'Check for argument "--resource-group"');
+    ok(grep(/--name Od_Kuka_do_Kuka/, @calls), 'Check for argument "--name"');
+    ok(grep(/--source Harvepino/, @calls), 'Check for argument "--source"');
+
+    az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka', size_gb => '42');
+    note("CMD: " . join(' ', @calls));
+    ok(grep(/--size-gb 42/, @calls), 'Check for argument "--size-gb"');
+};
+
+subtest '[az_disk_create] Create empty disk defining size' => sub {
+    my $azcli = Test::MockModule->new('sles4sap::azure_cli', no_auto => 1);
+    my @calls;
+    $azcli->redefine(assert_script_run => sub { @calls = $_[0]; return; });
+
+    az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka', size_gb => '42');
+    note("CMD: " . join(' ', @calls));
+    ok(grep(/--size-gb 42/, @calls), 'Check for argument "--size-gb"');
+};
+
+
+subtest '[az_disk_create] Check exceptions' => sub {
+    my $azcli = Test::MockModule->new('sles4sap::azure_cli', no_auto => 1);
+    my @calls;
+    $azcli->redefine(assert_script_run => sub { @calls = $_[0]; return; });
+
+    az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka', size_gb => '42');
```

Comment:
This test seems same of the previous one

Reply:
Ah sorry.. it was supposed to test the exceptions. I just pushed the fixed code: 
```subtest '[az_disk_create] Check exceptions' => sub {
    dies_ok{az_disk_create(resource_group => 'Pa_a_Pi', size_gb => '42')} "Croak with missing mandatory argument 'resource_group'";
    dies_ok{az_disk_create(name => 'Od_Kuka_do_Kuka', size_gb => '42')} "Croak with missing mandatory argument 'name'";
    dies_ok{az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka')} "Croak with missing mandatory argument 'size_gb'";
    dies_ok{az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka')} "Croak with missing mandatory argument 'source'";
    dies_ok{az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka', size_gb => '42', source => 'Slovenska_televizia')} "Croak with both 'size_gb' and 'source' defined at the same time";
};```